codegen/templates/tchannel_client.tmpl

```diff
@@ -0,0 +1,231 @@
+{{- /* template to render edge gateway http client code */ -}}
+// Code generated by zanzibar
+// @generated
+
+// Package {{.PackageName}} is generated code used to make or handle TChannel calls using Thrift.
+package {{.PackageName}}
+
+import (
+	"context"
+	"errors"
+	"strconv"
+	"time"
+
+	"github.com/uber/zanzibar/runtime"
+
+	{{range $idx, $pkg := .IncludedPackages -}}
+	{{$pkg.AliasName}} "{{$pkg.PackageName}}"
+	{{end}}
+)
+
+
+// Interfaces for the service and client for the services defined in the IDL.
+{{/* TODO: case */}}
+{{$clientID := .ClientID -}}
+{{$clientName := title .ClientID | printf "%sClient" -}}
+{{range $svc := .Services}}
+// TChan{{title $clientID}} is the interface that defines the server handler and client interface.
+type TChan{{title $clientID}} interface {
```

Comment:
The test helper uses it, e.g. https://github.com/uber/zanzibar/pull/56/files#diff-c89f3c36e508e3ece8aa23fcd9e8e592R28